doc/website/release-notes/iceoryx-unreleased.md

```diff
@@ -1,8 +1,8 @@
-# iceoryx vx.x.x
+# iceoryx v3.0.0 (UNRELEASED)
 
-## [vx.x.x](https://github.com/eclipse-iceoryx/iceoryx/tree/vx.x.x) (xxxx-xx-xx) <!--NOLINT remove this when tag is set-->
+## [v3.0.0](https://github.com/eclipse-iceoryx/iceoryx/tree/vx.x.x) (2025-03-16)
```

Comment:
This should not have been merged. Sorry, but please do not use 3.0 version numbers before it is not set in the `VERSION` file

Reply:
@elBoberido I don't understand why that is an issue as we are shortly before releasing `v3.0.0`.  It is also clear from the changes in `iceoryx-unreleased.md` that these will be API-breaking changes according to [SemVer](https://semver.org/). Anyway, I reverted the change on #2419

Reply:
Well, we just don't know whether we are shortly before releasing. it might be next month or in half a year. If someone stumbles upon this document and spreads the date on, e.g. reddit, then there is damage done when we do not release. Having the date and release numbers set now, creates expectations.